python/nano/src/bigdl/nano/pytorch/inference/optimizer.py

```diff
@@ -220,7 +342,7 @@ def _openvino_checker():
     '''
     check if openvino-dev is installed
     '''
-    return not find_spec("openvino-dev") is None
+    return not find_spec("openvino") is None
```

Comment:
I think we still need `openvino-dev` for pot?

Reply:
done